### PR TITLE
Add description to shipping method

### PR DESCRIPTION
**Pull request by IKarbowiak**
- Extend the `ShippingMethod` model with `description`.
- Extend the `ShippingMethod` type with `description` field.
- Extend mutation to add/edit description: `shippingPriceUpdate`, `shippingPriceCreate`.
- Support translating `ShippingMethod` `description`.

# Impact

* [x] New migrations
* [x] New/Updated API fields or mutations
* [ ] Deprecated API fields or mutations
* [ ] Removed API types, fields, or mutations
* [ ] Documentation needs to be updated

# Pull Request Checklist

* [ ] Privileged queries and mutations are guarded by proper permission checks
* [ ] Database queries are optimized and the number of queries is constant
* [ ] Database migration files are up to date
* [x] The changes are tested
* [x] GraphQL schema and type definitions are up to date
* [x] Changes are mentioned in the changelog

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-2821 Add description to shipping method](https://app.clickup.com/t/2549495/SALEOR-2821)

**Review by d-wysocki — APPROVED** [review]

**Review by maarcingebala — APPROVED** [review]

**Comment by maarcingebala** [comment]
Needs to wait for dashboard changes.

File: CHANGELOG.md [hunk 1]
```diff
@@ -43,6 +43,7 @@ All notable, unreleased changes to this project will be documented in this file.
 - Update also draft order line total price after getting the unit price from plugin - #7080 by @IKarbowiak
 - Fix failing product tasks when instances are removed - #7092 by @IKarbowiak
 - Update GraphQL endpoint to only match exactly `/graphql/` without trailing characters - #7117 by @IKarbowiak
+- Add description to shipping method - #7116 by @IKarbowiak
```
Comment (korycins) [inline CHANGELOG.md:46]: It looks like, we are changing an API a little bit here. Don't we want to add more details to changelog? Also, shouldn't be placed in breaking changes (I see that some types of input have been renamed/extended?

File: saleor/graphql/translations/mutations.py [hunk 2]
```diff
@@ -53,6 +53,10 @@ class TranslationInput(NameTranslationInput, SeoTranslationInput):
     description = graphene.JSONString()
 
 
+class DescriptionTranslationInput(NameTranslationInput):
+    description = graphene.JSONString()
```
Comment (fowczarek) [inline saleor/graphql/translations/mutations.py:57]: Maybe?
```suggestion
    description = graphene.JSONString(description="Translated shipping method description (JSON).")
```

**Review by fowczarek — APPROVED** [review]

**Review by jwm0 — CHANGES_REQUESTED** [review]
Missing `description` field on `ShippingMethodTranslatableContent`

File: saleor/graphql/schema.graphql [hunk 3]
```diff
@@ -2632,7 +2637,7 @@ type Mutation {
   shippingPriceDelete(id: ID!): ShippingPriceDelete
   shippingPriceBulkDelete(ids: [ID]!): ShippingPriceBulkDelete
   shippingPriceUpdate(id: ID!, input: ShippingPriceInput!): ShippingPriceUpdate
-  shippingPriceTranslate(id: ID!, input: NameTranslationInput!, languageCode: LanguageCodeEnum!): ShippingPriceTranslate
+  shippingPriceTranslate(id: ID!, input: DescriptionTranslationInput!, languageCode: LanguageCodeEnum!): ShippingPriceTranslate
```
Comment (jwm0) [inline saleor/graphql/schema.graphql:2640]: `DescriptionTranslationInput` name does not make sense, I suggest `ShippingPriceTranslationInput`

**Review by jwm0 — CHANGES_REQUESTED** [review]

**Review by jwm0 — APPROVED** [review]

**Pull request merged**